lib/project_types/script/commands/enable.rb

```diff
@@ -32,6 +32,12 @@ def call(_args, _name)
           title: project.script_name
         ))
         @ctx.puts(@ctx.message('script.enable.info'))
+
+      rescue Errno::ENOENT, Psych::SyntaxError
```

Comment:
Could we move this rescue into `acquire_configuration`, and raise `Errors::InvalidConfigYAMLError` there? I think its best to limit the domain of a rescue, and it seems more in line with our current error handling pattern. What do you think?